src/makeRobotAccountant.js

```diff
@@ -6,7 +6,27 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let currentNo = 1;
```

Comment:
```suggestion
  let currentNo = 0;
```
Чому 1? Було б логічніше починати з нуля і перевіряти if (no < 3 ....), що буде більш зрозуміло відповідно до умов задачі